src/docs/issue-platform.mdx

```diff
@@ -29,6 +29,8 @@ You may also set these optional attributes:
 - `noise_config`: A `NoiseConfig` that allows you to set policy on how many times a fingerprint must occur in a period of time before an issue is created. NoiseConfig has two attributes:
   - `ignore_limit`: How many occurrences need to occur before we create a new issue
   - `expiry_time`: The period of time the occurrences must occur in before we reset the count
+- `enable_auto_resolve`: Whether this issue type can be resolved automatically after 14 days of silence. By default this attribute is set to true.
```

Comment:
"14 days of no new occurrences"?

Reply:
Sorry, just used what was one the one [dev doc](https://docs.sentry.io/product/alerts/best-practices/#:~:text=In%20particular%2C%20regressions%20will%20be%20more%20common%20than%20you%20expect%20because%20Sentry%20auto%2Dresolves%20issues%20after%2014%20days%20of%20silence%20(configurable)%2C%20and%20many%20issues%20keep%20coming%20back%20after%20the%2014%2Dday%20window.)